main.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css">
+  <link rel="stylesheet" type="text/css" href='https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css'>
+  <link rel="stylesheet" href="/styles/home.css">
+  <link rel="stylesheet" href="/styles/header.css">
+  <title>판다마켓</title>
+</head>
+<body>
+  <header>
+    <!--로고    로그인버튼-->
+    <a href="/main.html" alt="판다마켓 메인"><img src="/images/panda-market-logo.png" id="panda-logo" alt="판다 마켓 로고"></a>
```

Comment:
```suggestion
    <a href="/main.html"><img src="/images/panda-market-logo.png" id="panda-logo" alt="판다 마켓 로고"></a>
```
alt는 img태그에 들어가게 됩니다!